packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.html

```diff
@@ -557,6 +557,19 @@
                     </oui-action-menu>
                 </oui-tile-definition>
 
+                <!-- Gateway IP-->
+                <oui-tile-definition
+                    term="{{:: 'server_configuration_gateway_ip' | translate }}"
+                >
+                    <oui-tile-description>
+                        <oui-clipboard
+                            class="ip-clipboard"
```

Comment:
Question: is the class "ip-clipboard" defined? is it required?

Reply:
yes, the class is already defined and it is required as well.